packages/visual-dev/src/components/Graphics.tsx

```diff
@@ -31,3 +31,35 @@ export const DataGraphic = (
IIPXwWavYRe//xSwMcEclmMCbu6OkW2LJ8tcThPgtHv4xJNnzUJjtokd+EAAAiGSURBVEYnysuqkJmZjusXTNfbpIgyc84UVNv933doOmbNjv2Zhd5AwHs6lx863/7vwk/AfHNHn0mgZ3739bG/6+4cZQ7nTzTGD8JhTyxBAAZZzJCN0btwpTcwMxrcHjjc3u537iccKWrEqYtO1NR7kZJkxKhMGXMnJyMhQXLJSZYh6UQOvWwLy/Ra2oC0uwjSXxDQByE7QKuDETkAGGD9L4J+F0IvGEC1S8UVxQ1fDDheBIPi9eFykxpXIn9nSw0+338FFdVuuD0aqq+4sa/AjjXrK+FQtRf1FDkQ5tVr33vhXIqqadMgwT4eo48/8QT1aC6lxOH6/2Dt38NpU3c0unw4WtSI5CQDJufqH/NMNhogGyTIRgMSYsRTjJmh+DQoXh8Uny8qQy1HksOFjdh04EqnnycYpLW/Xp33QB+a1I6o8tCoZE70OFwHGTy2+73Dwxsbq3Cppnl55bLrBmBSbnhSEIUDAxESJIJRkmCUCAZ/zHkJBInQEgM9ggtCrsZM15ih+ePW+zSG1x9D3atpUb9yK9K8vqEKVbXN95DJKCE/x4bCMiccTc0tGiK4UweMtf3oXgoueH4EiCpf9wwiZ5mi3M5u314AEU/OVWf3XBM5ANidfZthtDt8zPD5uFvXUGr9nwDq5fOb/b5bVxMjCIKjtr6l4TpttA0PLMrAzoJ6/GNbs78/M0wN9WfHADipk4nRt3otS5aLDJJhFYjCElmjK4pKAsMXj8wMLfOIXlwVpgb4M5Zwr/40vy+yEHnPSElsqS9Plzqx53RDQLZfEHGKFeHLrNkLoqpGv8qwJPmzEqfzBmj0LphD9g3debQBR4oaMSDFiLxsK/JGWGGzGlBY3LIWPMlqwND08KcLFvR/Rg6VUWtvrtUbnF6s3RLoFETArh89PKZvk+W1ISqFDgDDExP3Vzuds1waXgFjaW/P0+jSsOdE8zWuqHajotqNLYfqMTgtATWtmlxjh1tDN1oQl8zLT8HZMhcaGjuYZSBSDJBW971VgURd07016YmJFcNticsMkmEpEdpHTwwC2USwdJBt9fKVwAkBZnSYllkg6A6TSVLuvSn9T0YDvQuiVjcR7ZEkaeZvvjZGlzhxrYmqUffuKHe6p2ma5w4GVhLTWAYHNUReZ/dgf0EjzpY1QVG7Htgamm5CXrYVE3MTIZui+jko0Aki0hhcBdA2EN5LSrR8PICoAQC+81aphezKOJPFdPH5h0Z0PufWx8SU0NtSy5ysqGome6WgYgorXhg+3Xt5dvEl5S6Xy3tTV/sOtBnx7TuHIUamsgXdwCQ9b/fR30M6B1iD5KvNslgqKaDmjn6ito8eDAOJ7AB6Oshx8OkXTg8CcE3ocgJB8QSONdc6vGh0eTEgKaYvkQAAER0ym00/SI4xcYaT+LyLGddiyBskuvzDh0akn6lw0bELThSUNKFJ1TAuyyJE3g8gIpYk6RuxVgOHm7i7k599oXCcV+PJV99rTG9IBGveUHl13lAZ2ryBaHT5kGztX4tM4piXTCZT++D5cUbc9UC9oJWt30uS9A+z2fwc0Bz9QyIIkfcfqv2/bdwTd0InaJ6W13T+N1/N3U1EdUT0iJ4RQAThhYjY/5uKbI+IQ6HnZBv+QiT9nojeMxgN914VtyzLG5n5eb3tE4QHZn5eluWNetsRLcT09Fq4YWaDoihbACzQ2xZBSOyQZXlRvA/AtUYIvQ3MPExRlCMAoishuSBYamRZnkpEXSd8izPiruneHURUTkSPiv567OHvlz8qRN4eIfQOkGV5PRHFXfy6WIeIfiDL8nq97YhGRNO9CxRF+S9mfkpvOwTdQ0R/kGX5W3rbEa0IoXcBM5OiKG8AuF9vWwRdslaW5QdFd6tzhNC7gZlNqqp+yJ2EsxboCxF9ajabbyPSLx5bLCCEHgTMnKQoymYAM/W2RRDAAVmWFxNRo96GRDtiMC4IiKhRluUVAA7obYvgGgdkWV4hRB4cQuhBQkQ1/trjU71tiXeI6FP/b1Gjty2xghB6DyCiRrPZfBuAtXrbEses9ffJRU3eA4TQewgRuf0jvH/Q25Z4wz+F9qAYeOs5Qui9gIhYluVvSZL0fTGlE3n8wSO+L8vyt8T17h1i1D1EFEVZwcxrIHzjI0UNET0qPN5CQwg9DDBzlqIob0Ksegs3O2RZvl/4roeOaLqHASIqk2V5EYBfiKZl6Piv4S/8S02FyMOAqNHDjKIoy5j5NQDpetsSo1QT0SMiaER4EUKPAMw8QFXV5wF8hZnFNQ4Cfy3+stls/n8i/FP4ETdhBHG73XM0TfszM0/X25ZohogOSZL0DRGtNXIIoUcYZjaoqvp1AD9l5hS97YkmqDmN0Q/MZvNfRNinyCKE3kcw8xBVVX8M4DFmNuttj54QkQrgb2az+YdEVKW3PfGAEHofw8xDVVX9LoDVzBxXuZqJqAnAi2az+VdEVKG3PfGEELpOMHO6oihPE9E3mTlZb3siCRHZmfmPsiz/noiq9bYnHhFC1xlmTnG73U8w8xeZeYLe9oQTIiogoldNJtML/v64QCeE0KMIt9s9Q9O0R5n5AcTuPHw1Eb0pSdIak8l0UG9jBM0IoUchzGxUVXU5gEcBLGdmm942dQUROQBsALDGbDZvICKv3jYJAhFCj3KY2ej1emdpmraYmZcAmM/Msp42EZECYCcRbZIkabPRaNwvxB3dCKHHGMxsVhRlLoBFRJQPYCyA0cxsikR5/rXfZwEUMvMxAFtkWd7tnyITxAhC6P0Av1POSDSL/urfIAA2ZrYRUVKr1zb/MQ5/k9vBzI1XX6M5fXQRgEIAhWaz+aJwZhEIBAKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCAQCAQCgUAgEAgEAoFAIBAIdOP/AGQBsyAFwcunAAAAAElFTkSuQmCC"
```

Comment:
Are these graphics needed? I don't seem them being used anywhere, also we might be able to delete the enormous string of characters being passed to the `xlinkHref` attribute

Reply:
Confirmed not needed and removed